crates/core/src/network/connection_manager.rs

```diff
@@ -527,7 +527,7 @@ impl PeerRouting for ConnectionManager {
             let len_before = connections.len();
             connections.retain(|c| c != connection && !c.is_closed());
             if len_before > 0 && connections.is_empty() {
-                info!(%peer, "All registered connections to this node were terminated");
+                debug!(%peer, "All registered connections to this node were terminated");
```

Comment:
This should be logged exactly once on shutdown, did you find this to be noisy?

Reply:
Not noisy. I'll revert it.